src/bootlog.rs

```diff
@@ -9,10 +9,24 @@ use BOOTLOG_LIST;
 pub fn run(appconfig: &AppConfig) {
     let mut rng = thread_rng();
     let num_lines = rng.gen_range(50, 200);
+    let mut burst_mode = false;
+    let mut count_burst_lines = 0;
 
     for _ in 1..num_lines {
         let choice = rng.choose(&BOOTLOG_LIST).unwrap_or(&"");
-        let sleep_length = rng.gen_range(10, 1000);
+        let mut line_sleep_length = rng.gen_range(10, 1000);
+        let mut char_sleep_length = 5;
+        let burst_lines = rng.gen_range(5, 25);
+
+        if burst_mode && count_burst_lines < burst_lines {
+            line_sleep_length = 1;
```

Comment:
Sleep for 30ms here instead of 1ms because it gives a nice effect.

Reply:
Good job on spotting the two changes. I agree that it gives it a better effect.